src/assets/translations/en/main.json

```diff
@@ -2855,7 +2855,12 @@
       "claim": "Claim",
       "claimed": "Claimed",
       "discountText": "*%{percent}% off trade fees",
-      "noClaims": "No claims available"
+      "noClaims": "No claims available",
+      "banner": {
+        "title": "Put a hat on it.",
+        "description": "The more FOX or FOXwifhat you hold, the more tokens you get per swap.",
```

Comment:
Preferably blocking: should we use "FOX Wif Hat" or "FWH" here?
I don't think "FOXwifhat" is either the name or the ticker.

Reply:
I've updated the copy to use the ticker, and while I was looking around I noticed we called it the same thing on the FOX Ecosystem page. So I went ahead and updated that as well.